src/helpers/apiCalls.js

```diff
@@ -51,10 +51,7 @@ export const postNewJob = async (newJob, id) => {
 
 export const updateJobStatus = (id, jobId, newStatus) => {
   return fetch(`https://lienflash-be.herokuapp.com/api/v1/user/${id}/jobs/${jobId}?status=${newStatus}`, {
-    method: 'PATCH',
-    headers: {
-      'Content-Type': 'application/json'
-    }
+    method: 'PATCH'
```

Comment:
Did this work better?

Reply:
Yeah I did some research and it doesn't seem to make a difference to have the header in the patch requests.